.licenserc.yaml

```diff
@@ -70,6 +70,9 @@ header:
     - '**/mockito-extensions/**'
     - 'oap-server/server-library/library-async-profiler-jfr-parser'
     - 'docs/en/changes/changes.tpl'
+    - 'oap-server/server-core/src/main/proto/profile.proto'
+    - 'oap-server/server-receiver-plugin/skywalking-profile-receiver-plugin/src/main/proto/profile.proto'
+    - 'test/e2e-v2/java-test-service/e2e-service-provider/src/main/proto/profile.proto'
```

Comment:
Why skip all these files' license check? I can't find these files in the change logs.